front/pages/poke/templates/[tId].tsx

```diff
@@ -247,6 +248,37 @@ function TemplatesPage({
     [assistantTemplate, sendNotification, setIsSubmitting, router]
   );
 
+  const { submit: onDelete } = useSubmitFunction(async () => {
+    if (assistantTemplate === null) {
+      throw "unreachable";
```

Comment:
Can you wrap in error ? even if unreachable I'd say let's never throw strings

Reply:
will do! 👍🏻